app/helpers/issues_helper.rb

```diff
@@ -0,0 +1,21 @@
+module IssuesHelper
+  def contextualized_form_path(local_assigns)
```

Comment:
When we defined this implementation the idea was use the `contextualized_issue_path` name to open the form and the route the follow `p/:id/issues/:issue_id`. But it's OK. Maybe it's be better to refactor it in another PR.